File: WebLeave/SPA/src/app/_core/models/report/report-show/export-excel-date-dto.ts
import { ReportDateDetail } from './report-date-detail.model';

export interface ExportExcelDateDto {
  employeeNumber: string;
  no: string;
  employeeName: string;
  partCode: string;
  employeePostition: string;
  leaveType: string;
  time_Start: string;
  time_Of_Leave: string;
  status: string;
  fromDate: string;
  endDate: string;
  reportDateDetailDTO: ReportDateDetail;
}

File: WebLeave/API/Dtos/Report/ReportShow/ExportExcelDateDto.cs
namespace API.Dtos.Report.ReportShow
{
    public class ExportExcelDateDto
    {
        public string EmployeeNumber { get; set; }
        public string No { get; set; }
        public string EmployeeName { get; set; }
        public string PartCode { get; set; }
        public string EmployeePostition { get; set; }
        public string LeaveType { get; set; }
        public string Time_Start { get; set; }
        public string Time_Of_Leave { get; set; }
        public string Status { get; set; }
        public string FromDate { get; set; }
        public string EndDate { get; set; }
        public ReportDateDetailDTO ReportDateDetailDTO { get; set; }
    }
}